packages/protocol-sdk/src/premint/premint-client.test.ts

```diff
@@ -149,21 +149,34 @@ describe("ZoraCreator1155Premint", () => {
       });
       premintClient.apiClient.postSignature = vi.fn();
 
-      const premint = await premintClient.executePremintWithWallet({
+      const { request } = await premintClient.executePremint({
+        account: deployerAccount!,
         data: await premintClient.getPremintData({
```

Comment:
im realizing now that this name could be better `getPremintData` - "data" can mean anything what does it refer to in this case? signature and premint?